src/components/HrViewElements/HrPagination.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import {IoIosArrowBack, IoIosArrowForward} from "react-icons/io";
+
+type Props = {
+    currentPageNr: number,
+    totalPagesNr: number,
+    maxStudentsPerPage: number,
+    nextPage: () => void,
+    prevPage: () => void,
+    setMaxPerPage: (e: React.ChangeEvent<HTMLSelectElement>) => void
+}
+
+export const HrPagination = ({currentPageNr, prevPage, setMaxPerPage, nextPage, totalPagesNr, maxStudentsPerPage}: Props) => {
+    return <div className="flex flex-row gap-5 items-center">
+            Ilość elementów
+            <select onChange={(e) => setMaxPerPage(e)}
+                    className="select bg-white text-base-100 select-sm w-13">
+                <option disabled selected style={{display: "none"}}>{maxStudentsPerPage}</option>
+                <option>10</option>
```

Comment:
Sugeruję wszystkie takie miejsca generować, a nie wypisywać z palca. Z palca to można 2, max 3 rzeczy podobne wypisać.